tools/cldr-code/src/main/java/org/unicode/cldr/util/CLDRPaths.java

```diff
@@ -34,105 +33,211 @@ public class CLDRPaths {
+        )
+    );
+    public static final String TMP2_DIRECTORY = CldrUtility.getPath(
+        CldrUtility.getProperty(
+            "CLDR_TMP_DIR",
+            CldrUtility.getPath(LOCAL_DIRECTORY, "../cldr-tmp2/")
+        )
+    );
 
     /**
      * @deprecated please use XMLFile and CLDRFILE getSupplementalDirectory()
      * @see DEFAULT_SUPPLEMENTAL_DIRECTORY
      */
     @Deprecated
-    public static final String SUPPLEMENTAL_DIRECTORY = CldrUtility.getPath(COMMON_DIRECTORY, "supplemental/");
+    public static final String SUPPLEMENTAL_DIRECTORY = CldrUtility.getPath(
+        COMMON_DIRECTORY,
+        "supplemental/"
+    );
+
     /**
      * Only the default, if no other directory is specified.
      */
-    public static final String DEFAULT_SUPPLEMENTAL_DIRECTORY = CldrUtility.getPath(COMMON_DIRECTORY, "supplemental/");
+    public static final String DEFAULT_SUPPLEMENTAL_DIRECTORY = CldrUtility.getPath(
+        COMMON_DIRECTORY,
+        "supplemental/"
+    );
     public static final String LOG_DIRECTORY = CldrUtility.getPath(TMP_DIRECTORY, "logs/");
     public static final String TEST_DIR = CldrUtility.getPath(CLDRPaths.TMP_DIRECTORY, "test/");
 
     /** If the generated BAT files are to work, this needs to be set right */
-    public static final String COMPARE_PROGRAM = "\"C:\\Program Files (x86)\\Compare It!\\wincmp3.exe\"";
+    public static final String COMPARE_PROGRAM =
+        "\"C:\\Program Files (x86)\\Compare It!\\wincmp3.exe\"";
 
     /**
      * @deprecated Don't use this from any code that is run from the .JAR (SurveyTool, tests, etc).
      *             If it must be used, add a comment next to the usage to explain why it is needed.
      */
     @Deprecated
-    public static final String UTIL_DATA_DIR = FileReaders.getRelativeFileName(CldrUtility.class, "data/");
+    public static final String UTIL_DATA_DIR = FileReaders.getRelativeFileName(
+        CldrUtility.class,
+        "data/"
+    );
 
     public enum DIRECTORIES {
-        common_dtd, common_properties, common_uca,
+        common_dtd,
+        common_properties,
+        common_uca,
 
         common_bcp47(DtdType.ldmlBCP47),
 
-        common_annotations(DtdType.ldml), common_casing(DtdType.ldml), common_collation(DtdType.ldml), common_main(DtdType.ldml), common_rbnf(
-            DtdType.ldml), common_segments(DtdType.ldml), common_subdivisions(DtdType.ldml),
-
-        common_supplemental(DtdType.supplementalData), common_transforms(DtdType.supplementalData), common_validity(DtdType.supplementalData),
-
-        keyboards_android(DtdType.keyboard, DtdType.platform), keyboards_chromeos(DtdType.keyboard, DtdType.platform), keyboards_dtd(DtdType.keyboard,
-            DtdType.platform), keyboards_osx(DtdType.keyboard,
-                DtdType.platform), keyboards_und(DtdType.keyboard, DtdType.platform), keyboards_windows(DtdType.keyboard, DtdType.platform),
-                ;
+        common_annotations(DtdType.ldml),
```

Comment:
This is much better. The jumbled structure was from an earlier application of a formatter that arbitrarily joined lines.

Reply:
this illustrates why we might need to compromise; the formatter has to be allowed to change line-breaking in order to fix this

maybe there are formatters that will break lines but won't join them, I don't know